kotsadm/pkg/policy/policies.go

```diff
@@ -0,0 +1,121 @@
+package policy
+
+var (
+	ActionRead  = "read"
+	ActionWrite = "write"
+)
+
+// Redactor
+
+var (
+	RedactorRead  = Must(NewPolicy(ActionRead, "redactor."))
+	RedactorWrite = Must(NewPolicy(ActionWrite, "redactor."))
+)
+
+// Registry
+
+var (
+	RegistryRead = Must(NewPolicy(ActionRead, "registry."))
+)
+
+// Snapshots
+
+var (
+	BackupRead            = Must(NewPolicy(ActionRead, "backup."))
+	BackupWrite           = Must(NewPolicy(ActionWrite, "backup."))
+	RestoreRead           = Must(NewPolicy(ActionRead, "restore."))
+	RestoreWrite          = Must(NewPolicy(ActionWrite, "restore."))
+	SnapshotsettingsRead  = Must(NewPolicy(ActionRead, "snapshotsettings."))
+	SnapshotsettingsWrite = Must(NewPolicy(ActionWrite, "snapshotsettings."))
+)
+
+// Cluster
+
+var (
+	ClusterRead  = Must(NewPolicy(ActionRead, "cluster."))
+	ClusterWrite = Must(NewPolicy(ActionWrite, "cluster."))
+)
+
+// Gitops
+
+var (
+	GitopsRead  = Must(NewPolicy(ActionRead, "gitops."))
+	GitopsWrite = Must(NewPolicy(ActionWrite, "gitops."))
+)
+
+// Prometheus
+
+var (
+	PrometheussettingsWrite = Must(NewPolicy(ActionWrite, "prometheussettings."))
+)
+
+// App
+
+var (
+	AppList   = Must(NewPolicy(ActionRead, "app."))
+	AppRead   = Must(NewPolicy(ActionRead, "app.{{.appSlug}}"))
+	AppCreate = Must(NewPolicy(ActionWrite, "app."))
+	AppUpdate = Must(NewPolicy(ActionWrite, "app.{{.appSlug}}", appSlugFromAppIDGetter))
+)
+
+// App supportbundle
+
+var (
+	AppSupportbundleRead  = Must(NewPolicy(ActionRead, "app.{{.appSlug}}.supportbundle.", appSlugFromSupportbundleGetter))
```

Comment:
`AppSupportBundleRead`

Reply:
im going with case to match dot separators